visualization/app/codeCharta/core/download/download.service.ts

```diff
@@ -26,8 +26,8 @@ export class DownloadService {
         return fileName;
     }
 
-    private removeVisibleAttribute(root: CodeMapNode) {
-        let copy = JSON.parse(JSON.stringify(root));
+    private removeVisibleAttribute(nodes: CodeMapNode) {
+        let copy = JSON.parse(JSON.stringify(nodes));
```

Comment:
check if we can find a type to avoid stringify->parse

Reply:
Thanks 👍 the `JSON.parse(JSON.stringify(nodes))` methods are also used to remove JSON `$hashkey` attributes -> updated functionName